web/src/ui/timeline/TimelineView.tsx

```diff
@@ -54,24 +105,11 @@ const TimelineView = () => {
 		const timelineView = timelineViewRef.current
 		roomCtx.scrolledToBottom = timelineView.scrollTop + timelineView.clientHeight + 1 >= timelineView.scrollHeight
```

Comment:
Is this and the associated `scrollIntoView` effect still the proper way to keep at bottom, or should it use the virtualizer's methods instead?

Reply:
The virtualiser is effectively just a block element in `.timeline-view`, and the scrollToIndex is primarily useful for elements that may have been unloaded. For scrolling to the bottom, scrolling to the last element isn't correct as there is padding at the bottom of the timeline list that isn't taken into account. From looking into the code, their methods seem to just call element.scrollTo under the hood anyway. 

TLDR: As far as I can tell, yes.